runtime/src/tasks/fifo/tasks-fifo.c

```diff
@@ -94,16 +85,10 @@ static volatile task_pool_p
                            task_pool_tail;     // tail of task pool
 
 static int                 queued_task_cnt;    // number of tasks in task pool
-static int                 blocked_thread_cnt; // number of threads that
-                                               //   cannot make progress
+static atomic_uint_least64_t
+                           blocked_task_cnt;   // number of blocked tasks
```

Comment:
Is `blocked_task_cnt` still used for anything? I guess it's exposed through `chpl_task_getNumBlockedTasks()`, but I wonder if we can/should consider removing that (as well as `chpl_task_getNumQueuedTasks()`)

Reply:
Leaving a note for the record: it turns out these are exposed indirectly, via `here.blockedTasks` and `here.queuedTasks`. We don't think these are used, but time is short. I'm going to merge this PR as it stands and we'll look into deprecating those in the next release.